src/query.ts

```diff
@@ -11,6 +11,7 @@ interface Repo {
   id: number;
   nwo: string;
   token: string;
```

Comment:
Can you make `token` and `pat` optional please. Because of the way that we're parsing the repos list with `const repos: Repo[] = JSON.parse(...)` whatever garbage object we get out of `JSON.parse` is just assumed to be a valid `Repo[]`. Any type safety only exists at compile time. So later on when we pass options to `downloadDatabase` we're actually passing an `undefined` but the code doesn't know this and happily lets us pass that as a non-optional argument.